implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/types/DynatraceDistributionSummary.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2022 VMware, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.dynatrace.types;
+
+import io.micrometer.core.instrument.AbstractMeter;
+import io.micrometer.core.instrument.DistributionSummary;
+import io.micrometer.core.instrument.distribution.HistogramSnapshot;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.TimeUnit;
+
+public class DynatraceDistributionSummary extends AbstractMeter implements DistributionSummary, DynatraceSummarySnapshotSupport {
```

Comment:
All of these implementations are public, so thats why I made it public.

Reply:
Hum, but if they are public it means other packages in this project can still use them, no? If it's something that just an implementation detail and the micrometer code uses the abstract class/interface, I wonder if this shouldn't be better suited for package private? 

Maybe not this one, but like the `DynatraceSummary` or `DynatraceSummarySnapshot` will only be used internally.

Reply:
The `DynatraceSummary` i can make package private, but not the Snapshot, otherwise ill not be able to use it in the `v2` package. I also do not want to put it in the `v2` package, since it isn't really specific to v2.

Reply:
Resolved for me :)
@joaopgrassi you can click the "Resolve conversation" button if it is revolved for you as well :)